### ISSUE TITLE
Add more supported iOS versions

### ISSUE THREAD

**Issue by johnnieWa1ker** [johnnieWa1ker/Transition#9]

**Comment by Gargo** [comment]
@johnnieWa1ker I added this package as local, downgraded minimum iOS version to 13.0 and removed everything related to `BottomSheetProps`

**Comment by johnnieWa1ker** [comment]
Yes, for local use this is a normal option. I had an idea to add different versions of support separately for `BottomSheetProps` and without it, but there was no time. I will deal with this in the coming days.
Thanks for highlighting the need 🤝
